classes/itembase.php

```diff
@@ -975,9 +975,9 @@ public function set_editor() {
         }
 
         $context = \context_module::instance($this->cm->id);
-        $editoroptions = array('trusttext' => true, 'subdirs' => true, 'maxfiles' => -1, 'context' => $context);
+        // I have to set 'trusttext' => false because 'noclean' is ignored if trusttext is enabled!
```

Comment:
Are you sure? Or I'm reading something wrong... or `$options[trusttext]` defaults to false:

https://github.com/stronk7/moodle/blob/master/lib/filelib.php#L144-L148

Please, check, ciao :-)

Reply:
I simply reported what I found in
https://github.com/moodle/moodle/blob/master/lib/filelib.php#L194

// noclean ignored if trusttext enabled

and this is the only way I found to add the tag a into the content of am label item of a surveypro.

Reply:
so, do you mean I should replace:

$editoroptions = array('noclean' => true, 'trusttext' => false, 'subdirs' => true, 'maxfiles' => -1, 'context' => $context);

with:

$editoroptions = array('noclean' => true, 'subdirs' => true, 'maxfiles' => -1, 'context' => $context);

Reply:
I think I just emended the code. Waiting for GHT.

Reply:
GHT: Good Ham Taking ? 🤣

(Sorry could not resist, and I was reading in another channel about hams, so the association came magically, hahaha)

Ciao 😄

Reply:
😄